src/app/shared/model/invalid/template.invalid.ts

```diff
@@ -0,0 +1,66 @@
+import { AudioVersionTemplateModel } from '../audio-version-template.model';
+import { AudioFileTemplateModel } from '../audio-file-template.model';
+import { BaseInvalid } from './base.invalid';
+
+/**
+ * Audio version template length
+ */
+const checkMinimum = (min, max): string => {
+  if (min === null && max === null) {
+    return null; // allow unset
+  } else if (min === null) {
+    return `Must set a minimum`;
+  } else if (isNaN(parseInt(min, 10))) {
+    return `Minimum is not a number`;
+  } else if (min < 1) {
+    return `Minimum must be greater than 0`;
+  } else if (min >= max) {
+    return 'Minimum must be less than maximum';
```

Comment:
Feel free to ignore this, but I wondered if we could not show the invalid message when the user is filling out the form in order and hasn't entered a maximum yet. Looks like they can have both be unset but if they set one, they have to set both. Is that just because that's how templates work? What if they want a min but no max?